AIPscan/static/js/tasks.js

```diff
@@ -17,29 +17,20 @@ function new_fetch_job(storageServiceId){
 }
 
 function package_list_task_status(taskId, showcount, fetchJobId){
-  const inspect_logs = ": Please inspect the celery logs"
   const status_pending = 'PENDING'
   const status_progress = 'IN PROGRESS'
-  const status_failure = "FAILURE"
   $.ajax({
     type: 'GET',
     url: '/aggregator/package_list_task_status/' + taskId,
     datatype: "json",
     success: function(data) {
-      state = data['state']
-      if (state != status_pending && state != status_progress && state != status_failure) {
+      state = data['state'];
```

Comment:
You may want to initialize `state` with `let state = ...`. Otherwise I think loose mode JS will make this a global variable and strict mode will throw an error.

Reply:
Ah, yep. Language switching failings.

Reply:
Yeah no worries, that happens!